src/test/java/com/wikia/webdriver/elements/common/Navigate.java

```diff
@@ -17,7 +23,7 @@ public Navigate toPage(String pageName) {
     String host = UrlBuilder.getHostForWiki();
     String cacheBuster = pageName.equals("") || pageName.equals("/") ? "" : "?cb=" + DateTime.now().getMillis();
 
-    driver.get("http://" + host + pageName + cacheBuster);
+    driver.get("http://" + host + pageName + cacheBuster + "&" + defaultTestingGroup);
```

Comment:
As it's only temporary change I used the quicker way to do it. In general that might be a good idea, but I'd like to investigate it when we will turn off experiment and delete that parameter from URL.